packages/mgt-spfx-utils/README.md

```diff
@@ -0,0 +1,154 @@
+# Utilities for SharePoint Framework Webparts using Microsoft Graph Toolkit
+
+[![npm](https://img.shields.io/npm/v/@microsoft/mgt-spfx-utils?style=for-the-badge)](https://www.npmjs.com/package/@microsoft/mgt-spfx-utils)
+
+![SPFx 1.15.2](https://img.shields.io/badge/SPFx-1.15.2-green.svg?style=for-the-badge)
+
+Helper functions to ease lazy loading of mgt components when using disambiguated web components in SharePoint Framework Webparts.
+
+## Installation
+
+To load lazy Microsoft Graph Toolkit components from the library, add the `@microsoft/mgt-spfx-utils` package and if using React, the `@microsoft/mgt-react` package as dependencies to your SharePoint Framework project:
+
+```bash
+npm install @microsoft/mgt-spfx-utils
+```
+
+or
+
+```bash
+yarn add @microsoft/mgt-spfx-utils
+```
+
+or
+
+```bash
+npm install @microsoft/mgt-spfx-utils @microsoft/mgt-react
+```
+
+or
+
+```bash
+yarn add @microsoft/mgt-spfx-utils @microsoft/mgt-react
+```
+
+> **Important:** Since the a given web component tag can only be registered once these approaches **must** be used along with the `customElementHelper.withDisambiguation('foo')` approach as this allows developers to create disambiguated tag names.
```

Comment:
```suggestion
> **Important:** Since a given web component tag can only be registered once these approaches **must** be used along with the `customElementHelper.withDisambiguation('foo')` approach as this allows developers to create disambiguated tag names.
```